### PR TITLE
Add requirements for Thrift connector

### PR DESCRIPTION
**Pull request by Jessie212**

### CODE REVIEW

File: docs/src/main/sphinx/connector/thrift.rst [hunk 1]
```diff
@@ -4,7 +4,9 @@ Thrift connector
 
 The Thrift connector makes it possible to integrate with external storage systems
 without a custom Trino connector implementation by using
-`Apache Thrift <https://thrift.apache.org/>`_ on these servers. It is therefore generic and can provide access any backend, as long as it exposes the expected API by using Thrift.
+`Apache Thrift <https://thrift.apache.org/>`_ on these servers. It is therefore
+generic and can provide access to any backend, as long as it exposes the expected
```
Comment (Jessie212) [inline docs/src/main/sphinx/connector/thrift.rst:8]: Format and grammar fix.

**Review by mosabua — APPROVED** [review]
Looks good.

File: docs/src/main/sphinx/connector/thrift.rst [hunk 2]
```diff
@@ -14,6 +16,13 @@ splits and data. The connector randomly chooses a server to talk to from the ava
 instances for metadata calls, or for data calls unless the splits include a list of addresses.
 All requests are assumed to be idempotent and can be retried freely among any server.
 
+Requirements
+------------
+
+To connect to your custom servers with the Thrift protocol, you need:
+
+* Network access from the Trino coordinator and workers to the Thrift servers.
```
Comment (hashhar) [inline docs/src/main/sphinx/connector/thrift.rst:24]: Also required is the step of implementing your own Thrift interface.

See https://github.com/trinodb/trino/pull/8377/files#diff-33aa44cbc9e5070dad299e4d6c9815bfc5b3eac7800ce6efa4f8453955a6c813R96-R102.

> Thrift IDL file
> ---------------
> The following IDL describes the ``TrinoThriftService`` that must be implemented:
> .. literalinclude:: /include/TrinoThriftService.thrift
>     :language: thrift

Comment (hashhar) [reply docs/src/main/sphinx/connector/thrift.rst:24]: May be worded as ```An implementation of ``TrinoThriftService`` which is described by the Thrift IDL at `/include/TrinoThriftService.thrift` ```

**Comment by Jessie212** [comment]
@hashhar Should be good now.

**Comment by mosabua** [comment]
ping @hashhar ?

**Review by hashhar — APPROVED** [review]

**Pull request merged**